wdfn-server/waterdata/templates/monitoring_location.html

```diff
@@ -78,48 +78,70 @@ <h1>{{ stations[0].station_nm }}</h1>
                     <p id="site-description">{{ components.Description(stations[0].site_no, location_with_values, parm_grp_summary) }}</p>
                 </div>
 
-                {{ components.TimeSeriesComponent(stations[0].site_no, stations[0].dec_lat_va, stations[0].dec_long_va) }}
+                {% if request.user_agent.browser == 'msie' %}
+                    {% block javascript %}
```

Comment:
I don't think a block is necessary here. Unless this block will be overridden somewhere else this should be removed.